salt/tests/unit/formulas/data/base_pillar.yaml

```diff
@@ -187,3 +187,4 @@ kubernetes:
 addons:
   dex:
     enabled: True
+oidc: {}
```

Comment:
Maybe worth adding a unit test with an OIDC in pillar content (just to ensure that we do not break the salt state rendering)